src/dotnet/commands/dotnet-new/VisualBasic_Console/Program.vb

```diff
@@ -0,0 +1,7 @@
+Imports System
+
+Module Program
```

Comment:
FYI I used `Program` instead of the default VB module name `Module1` for consistency with other templates